marshmallow_oneofschema/one_of_schema.py

```diff
@@ -56,6 +56,7 @@ def get_obj_type(self, obj):
     """
     type_field = 'type'
     type_field_remove = True
+    nest_result = False
```

Comment:
You could switch this to be a field name instead of a boolean. That way we can choose what field to nest under, rather than having to use `value`.
Could default to `None` to indicate no nesting.

Reply:
+1 to this. I made it use the value if `nest_result` is a str otherwise use `value`.